android/src/toga_android/widgets/switch.py

```diff
@@ -32,7 +32,6 @@ def set_text(self, text):
         # is required to get the text to truly **use** one single line!
         self.native.setSingleLine(False)
         self.native.setText(str(self.interface.text))
-        self.rehint()
```

Comment:
`rehint` calls in backend setters are redundant, because there's already a call in the core setter.